src/main/java/Duke.java

```diff
@@ -1,10 +1,191 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+/**
+ * This class is to build a personal assistance chat-bot called "Kaman"
```

Comment:
I like how you personalize your chatbot with a name. 

I think there is a minor wording issue with the phrase "personal assistance chat-bot" in your comment. Perhaps you might want to rephrase it to "personal assistant chatbot".